app/Http/Controllers/Backend/Export/ExportController.php

```diff
@@ -83,10 +83,14 @@ private static function exportJson(Carbon $begin, Carbon $end): JsonResponse {
      * @throws DataOverflowException
      */
     private static function exportPdf(Carbon $begin, Carbon $end): \Illuminate\Http\Response {
+        $statuses = ExportBackend::getExportableStatuses(auth()->user(), $begin, $end);
+
         return Pdf::loadView('pdf.export-template', [
-            'statuses' => ExportBackend::getExportableStatuses(auth()->user(), $begin, $end),
-            'begin'    => $begin,
-            'end'      => $end
+            'statuses'     => $statuses,
+            'begin'        => $begin,
+            'end'          => $end,
+            'sum_duration' => $statuses->reduce(fn($sum, $s) => $sum + $s->trainCheckin->duration),
+            'sum_distance' => $statuses->reduce(fn($sum, $s) => $sum + $s->trainCheckin->distance) / 1000,
```

Comment:
I think this is more readable.

```suggestion
            'sum_duration' => $statuses->sum('trainCheckin.duration'),
            'sum_distance' => $statuses->sum('trainCheckin.distance') / 1000,
```

Reply:
Maybe this can be also moved to the blade? In my opinion it is enough that we transfer `$statuses` from the Controller.

Reply:
Fixed with [dc482cc](https://github.com/Traewelling/traewelling/pull/1187/commits/dc482cc81292b04b0762c994575a929659035e36).

Reply:
I'd rather have that in the Controller, rather than in the blade.